<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidgetLeft" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidgetRight" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearSelections">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Selections</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCut">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut Graph</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radForeground">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>106</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Foreground</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radBackground">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>106</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Background</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>520</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram Bins Per Dimension:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHistogramBins">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>590</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldHistogramBins">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>610</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;</string>
    </property>
    <property name="title">
     <string>Lambda</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>LambdaMax</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLambdaMax">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>.05</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>% of LambdaMax</string>
     </property>
    </widget>
    <widget class="QSlider" name="sldLambda">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblLambda">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.025</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>64</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Lambda</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>637</x>
      <y>600</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Segmentation</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Color_Image"/>
    <addaction name="actionOpen_Grayscale_Image"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Color_Image">
   <property name="text">
    <string>Open As Color Image</string>
   </property>
  </action>
  <action name="actionOpen_Grayscale_Image">
   <property name="text">
    <string>Open As Grayscale Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sldHistogramBins</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblHistogramBins</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
